src/mesido/workflows/multicommodity_simulator_workflow.py

```diff
@@ -515,7 +515,8 @@ def energy_system_options(self):
 
         self.gas_network_settings["head_loss_option"] = HeadLossOption.LINEARIZED_N_LINES_EQUALITY
         self.gas_network_settings["network_type"] = NetworkSettings.NETWORK_TYPE_HYDROGEN
-        self.gas_network_settings["minimize_head_losses"] = True
+        self.gas_network_settings["minimize_head_losses"] = False
```

Comment:
I think we should add a head loss test case where self.gas_network_settings["minimize_head_losses"] = False. What do you think?

Reply:
I agree! then we are sure that it really works as intended. 
Ideally not in the multicommodity_simulator test but in the headloss test.

Reply:
I actually updated the test cases where the headloss is checked when HeadlossOption == LINEARIZED_N_LINES_EQUALITY. I set self.gas_network_settings["minimize_head_losses"] = False since Equality constraints should always be able to be used when the objective to minimize head losses is turned off.

Reply:
Nice